cylc/flow/cycling/iso8601.py

```diff
@@ -895,7 +895,24 @@ def get_dump_format():
 def get_point_relative(offset_string, base_point):
     """Create a point from offset_string applied to base_point."""
     try:
-        interval = ISO8601Interval(str(interval_parse(offset_string)))
+        if len(re.split('\\+|-', offset_string)) > 2:
```

Comment:
Actually there is no need to handle the case of more than one offset separately. I've opened a PR against this branch at https://github.com/markgrahamdawson/cylc-flow/pull/3